sample-apps/javascript-sample-app/request-metrics.js

```diff
@@ -0,0 +1,54 @@
+const TOTAL_BYTES_SENT_METRIC = 'totalBytesSent';
+const TOTAL_API_REQUESTS = 'apiRequests';
+const LATENCY_TIME = 'latencyTime';
+
+module.exports = (meter) => {
```

Comment:
Instead of getting the meter as parameter you could import the meter defined in meter.js directly, no?